src/lib.rs

```diff
@@ -18,7 +16,7 @@ pub use crate::production::Production;
 pub use crate::term::Term;
 
 #[cfg(feature = "ABNF")]
-pub use augmented::ABNF;
+pub use parsers::ABNF;
```

Comment:
fiddly thing, surprised clippy didn't spot but can `ABNF` be added to the `use` below with `Format, BNF`?

Reply:
ah no because it's under a feature flag ✨